Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm

```diff
@@ -166,6 +176,35 @@ void didReceiveAuthenticationChallenge(Ref<WebKit::AuthenticationChallengeProxy>
         [m_delegate.getAutoreleased() didReceiveAuthenticationChallenge:nsURLChallenge completionHandler:completionHandler.get()];
     }
 
+#if ENABLE(SCREEN_TIME)
+    void getScreenTimeURLs(std::optional<WTF::UUID> identifier, CompletionHandler<void(HashSet<URL>&&)>&& completionHandler) const final
+    {
+        if (![PAL::getSTWebHistoryClass() instancesRespondToSelector:@selector(fetchAllHistoryWithCompletionHandler:)])
+            return completionHandler({ });
+
+        STWebHistoryProfileIdentifier webHistoryProfileId = nil;
+        if (identifier)
+            webHistoryProfileId = identifier->toString();
+
+        RetainPtr webHistory = adoptNS([PAL::allocSTWebHistoryInstance() initWithProfileIdentifier:webHistoryProfileId]);
+
+        [webHistory fetchAllHistoryWithCompletionHandler:makeBlockPtr([completionHandler = WTFMove(completionHandler)] (NSSet<NSURL *> *urls, NSError *error) mutable {
```

Comment:
Style guide; no space between capture list and arg list.

https://webkit.org/code-style-guidelines/#spacing-lambda-paren

```suggestion
        [webHistory fetchAllHistoryWithCompletionHandler:makeBlockPtr([completionHandler = WTFMove(completionHandler)](NSSet<NSURL *> *urls, NSError *error) mutable {
```